README.md

```diff
@@ -1,15 +1,18 @@
 # OpenTelemetry Semantic Conventions
 
-Welcome to the new repository!
+[![Checks](https://github.com/open-telemetry/semantic-conventions/workflows/Checks/badge.svg?branch=main)](https://github.com/open-telemetry/semantic-conventions/actions?query=workflow%3A%22Checks%22+branch%3Amain)
+![GitHub tag (latest SemVer)](https://img.shields.io/github/tag/open-telemetry/semantic-conventions.svg?logo=opentelemetry&&color=f5a800&label=Latest%20release)
+![Specification Version](https://img.shields.io/badge/Specification_version-v1.26.0-blue?logo=opentelemetry&color=f5a800)
```

Comment:
```suggestion
[![OTel specification version](https://img.shields.io/badge/OTel_specification_version-v1.26.0-blue?logo=opentelemetry&color=f5a800)](https://github.com/open-telemetry/opentelemetry-specification/releases/tag/v1.26.0)
```
Let's add a link to the spec release.

We'll also need to add `https://github.com/open-telemetry/opentelemetry-specification/releases/tag/` to the patterns in https://github.com/open-telemetry/semantic-conventions/blob/main/internal/tools/update_specification_version.sh then.